README.md

```diff
@@ -10,6 +10,9 @@ If you need to update a submodule in this repository to point to a new commit ha
 
 For our own roles that are in this repository as submodules, they should be on the latest `master`. You can update them with `git submodule update --remote`.
 
+Warning: ansible-playbook will silently skip tasks in the roles defined as
+submodules if the submodules haven't been checked out. Be sure to run `git submodule update --init --recursive` to initialize the submodules.
+
```

Comment:
Since running `git submodule update --init --recursive` is actually step 1 for new clones of this repo, could you put this advice before the `git submodule update --remote` line above, like we do in ansible-secrets?

Reply:
Yep that makes sense, thanks. :+1: